docs/index.md

```diff
@@ -60,6 +60,29 @@ provider "ably" {
 }
 ```
 
+## Importing existing resources
+
+In order to import a resource, you need to add the resource to your Terraform configuration file, and then follow https://www.terraform.io/cli/import. 
+
+For example, in order to import an app named `ably-tf-provider-app-0000`, add something similar to the following:
+  
+  ```terraform
```

Comment:
Errant space?

Reply:
Fixed in a8102db37f39001c5a161a00f7d1347e2080ec1f